build-parent/pom.xml

```diff
@@ -1235,7 +1248,17 @@
                 </plugins>
             </build>
         </profile>
-
+        <profile>
+            <id>dependency-check</id>
```

Comment:
```suggestion
            <id>owasp-check</id>
```
Maybe?

Reply:
@aloubyansky Sure, makes sense to have a more specific name as `dependency check` can mean a few things